lib/src/Rating/Rating.tsx

```diff
@@ -1,20 +1,16 @@
-import React from "react";
-import ReactRating from "react-rating";
-import { SVGStar, SVGStarHollow } from "./RatingStar";
+import React, { useState, useEffect } from "react";
+import classnames from "classnames";
+import { SVGComponent } from "./SVGComponent";
+import "./rating.scss";
 
-export interface RatingProps {
-    className?: string;
+export type RatingProps = JSX.IntrinsicElements["input"] & {
+    initialValue?: number;
     colors?: [string, string];
-    disabled?: boolean;
     iconHeight?: number;
     iconWidth?: number;
-    id?: string;
-    initialValue?: number;
-    onChange?: (value: number) => void;
-    readOnly?: boolean;
-    tooltipList?: Array<string>;
-    useHollow?: boolean;
-}
+    className?: string;
```

Comment:
You don't need to define this, it's already part of `JSX.IntrinsicElements["input"]` interface
```suggestion
```

Reply:
Can you also add `wrapperProps` which extends `JSX.IntrinsicElements["div"]`. This is something we started adding to all our wrapped components so that users can pass wrapper props when necessary. This is especially useful if you are dealing with Bootstrap grid. Passing `col-5` to this component's `className` will not align it correctly since this className will go to the `input` not it's wrapper `div`

Reply:
removed. it and added wrapperProps